features/magnum/k8s-fragment-fixes.pan

```diff
@@ -0,0 +1,22 @@
+# Magnum fixes for supporting recent K8s versions
+# Fixes must be applied with patch command
+unique template features/magnum/k8s-fragment-fixes;
+
+include 'components/filecopy/config';
+# auto-scaling patch
+# panlint disable=LP006
+prefix '/software/components/filecopy/services/{/usr/lib/python3.9/site-packages/magnum/drivers/common/templates/kubernetes/fragments/enable-auto-scaling.sh.patch}';
+'config' = file_contents('features/magnum/magnum-k8s-fixes/enable-auto-scaling.sh.patch');
+'perms' = '0644';
+
+# cinder-csi patch
+# panlint disable=LP006
+prefix '/software/components/filecopy/services/{/usr/lib/python3.9/site-packages/magnum/drivers/common/templates/kubernetes/fragments/enable-cinder-csi.sh.patch}';
+'config' = file_contents('features/magnum/magnum-k8s-fixes/enable-cinder-csi.sh.patch');
+'perms' = '0644';
+
+# keystone-auth patch
+# panlint disable=LP006
+prefix '/software/components/filecopy/services/{/usr/lib/python3.9/site-packages/magnum/drivers/common/templates/kubernetes/fragments/enable-keystone-auth.sh.patch}';
```

Comment:
```suggestion

prefix '/software/components/filecopy/services';

prefix '{/usr/lib/python3.9/site-packages/magnum/drivers/common/templates/kubernetes/fragments/enable-auto-scaling.sh.patch}';
'config' = file_contents('features/magnum/magnum-k8s-fixes/enable-auto-scaling.sh.patch');
'perms' = '0644';

# cinder-csi patch
prefix '{/usr/lib/python3.9/site-packages/magnum/drivers/common/templates/kubernetes/fragments/enable-cinder-csi.sh.patch}';
'config' = file_contents('features/magnum/magnum-k8s-fixes/enable-cinder-csi.sh.patch');
'perms' = '0644';

# keystone-auth patch
prefix '{/usr/lib/python3.9/site-packages/magnum/drivers/common/templates/kubernetes/fragments/enable-keystone-auth.sh.patch}';
```

Reply:
Done. I was not aware that prefix could accept a relative path...